src/chrome/content/rules/Marcusoft.net.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+	Active mixed content:
+		- www.marcusoft.net
```

Comment:
I just found fonts and trackers, so we can safely ignore it.

Reply:
Fixed, thanks!